.github/workflows/rust.yml

```diff
@@ -29,8 +29,11 @@ jobs:
               AS_DEPENDENCY: true
               PIN_VERSIONS: true
     steps:
-      - name: Install test dependencies
+
+      - name: Install fuzz dependencies
```

Comment:
Can't this be dropped entirely now?

Reply:
yes, done in 9b6b50a9870608bff9e7ecf8042e83cdb09cf640